en/docs/observability/perform-root-cause-analysis.md

```diff
@@ -8,88 +8,94 @@ Performing root cause analysis is crucial in identifying and rectifying the unde
+### Step 1: Detect the anomalies
 
-With the sample data, you will observe spikes in the throughput graph. Let's see how you can find
-the root cause by analyzing this data.
+#### Analyze the throughput graph
+The throughput graph will show the throughputs of requests per hour for a selected timestamp. 
+
+With the sample data, you will observe spikes in the throughput graph.  You can find the root cause for this by learning how to analyze the throughput graph.
 
 Follow the steps below. 
 
-1. Hover over the throughput graph at 01/04/2021 13:41:00. You can observe that 46.06 errors had occurred at the time. 
+1. Hover over the throughput graph where you see spikes. You can observe the number of errors that occurred at the time. 
     
     !!!tip
         You can expand the graph by clicking and dragging the cursor over the period you want to drill down.
         
-2. Click on the graph on 01/04/2021 at 13:41:00.
+2. Click the graph on a high point.
 3. Observe the logs in the logs panel.
 4. Note multiple logs are mentioning a connection error with the hr-service.
  
 ![Analyzing the throughput graph](../assets/img/observability/throughput-graph-analysis.png)
  
-### Analyzing the latency graph
+#### Analyze the latency graph
 
 The latency graph shows the latencies of each request.
 
-With the sample data, you will observe spikes in the latency graph. By analyzing the latency graph, you can identify which API invocation caused the error.  Let's see how to find the
-root cause by analyzing this data.
+With the sample data, you will observe spikes in the latency graph. By analyzing the latency graph, you can identify the errors that caused the spikes and the exact API invocation(s) that caused the errors. 
 ![Analyzing the latency graph](../assets/img/observability/latency-graph-analysis.png)
 
-Follow the steps below.
+Follow the steps below to learn how to analyze the latency graph.
 
-1. Hover over the latency graph and click at 01/04/2021 13:41:00.
+1. Hover over the latency graph and click on a high point.
+   
+    !!!tip
+        You can expand the graph by clicking and dragging the cursor over the period you want to drill down.
+        
 2. Observe the latencies of each request listed in the low-code diagram.
 3. Click on one request shown on the low code diagram (latencies). You can observe the status code for that request and thereby identify the exact API invocation that caused any error(s).
 
-### Diagnostics View
-The Diagnostics view provides the capability to view errors, throughput, latency, CPU, memory, and logs simultaneously for a particular event. You can detect and analyze errors and anomalies in detail via the diagnostics view.
+### Step 2: Drill into the analytics
+The Diagnostics view provides the capability to view errors, throughputs, latencies, CPU, memory useage, and logs simultaneously for a particular event. You can detect and analyze errors and anomalies in detail via the diagnostics view.
```

Comment:
```suggestion
The Diagnostics view provides the capability to view errors, throughputs, latencies, CPU, memory usage, and logs simultaneously for a particular event. You can detect and analyze errors and anomalies in detail via the diagnostics view.
```

Reply:
Also if we use `usage` for memory  why not for CPU?

Reply:
When you say analyze CPU, it implies CPU usage right? But when you say memory, it can be memory dump as well, that was my thinking.  Do you think it might be vague when you say analyzing CPU?

Reply:
What goes to my mind was `errors, throughputs, latencies, CPU and memory usage, and logs`

Reply:
I am thinking if we put it that way, it would imply that we could view them together, where as we can actually view them separately.  But if it is vague, I'll reword CPU to CPU usage.